pkg/pki/k8scsrpki/k8scsr_user.go

```diff
@@ -344,3 +345,45 @@ func (c *k8sCSR) Approve(ctx context.Context, signingReq *certsigningreqv1.Certi
 
 	return nil
 }
+
+func (c *k8sCSR) getCAChain(ctx context.Context, signingReq *certsigningreqv1.CertificateSigningRequest, certs []*certutil.CertificateContainer) ([]byte, error) {
+	var caChain []byte
+	if strings.Split(signingReq.Spec.SignerName, "/")[0] == v1alpha1.CertManagerSignerNamePrefix {
+		clusterIssuer := &certv1.ClusterIssuer{}
+		clusterIssuerName := strings.Split(signingReq.Spec.SignerName, "/")[1]
```

Comment:
I think it may be worth having a variable instead of doing the string splitting twice. For me it would make the code look better, but it's very optional.

Reply:
Thanks for noticing